FirebaseAnalytics/README.md

```diff
@@ -1,2 +1,111 @@
-This directory open sources select files from the Firebase Analytics SDK. Note
```

Comment:
We probably still want something like these first two lines.

Reply:
GitHub isn't picking up the git mv very well. These two lines were moved to a README.md in the new Sources sub-directory: https://github.com/firebase/firebase-ios-sdk/pull/13056/files#diff-98fb7621db2b12dd007a7101be7f7de33b98663f976f76bec7c41f067f978b47